.eslintrc

```diff
@@ -24,7 +24,7 @@
     "no-process-env": 0,
     "no-inline-comments": 0,
     "line-comment-position": 0,
-    "no-warning-comments": 1
+    "no-warning-comments": 0
```

Comment:
This is `TODO` comments right? Not a fan?

Reply:
Am a Fan... Right now need... hmm.. I wonder why I set that 🤔 
Let me fix it when you're done with the review.